src/pseudo/load_psp.jl

```diff
@@ -1,6 +1,24 @@
+using PseudoPotentialData
```

Comment:
Can you avoid this dependency by making it an extmodule ?

Reply:
Good idea, I didn't think of it.